_example/hello-world/Makefile

```diff
@@ -10,16 +10,16 @@ pack:
 
 dbuild:
 	@docker run --rm \
-		-u ${shell id -u}:${shell id -g} \
```

Comment:
Yeah, I'm baffled myself with that.
I'm running the exact same code as the make `build` target.